src/Unity.Mathematics.Editor/PostNormalizedVectorDrawer.cs

```diff
@@ -139,20 +152,25 @@ protected virtual double4 Normalize(double4 value)
             return math.normalizesafe(value);
         }
 
-        public override float GetPropertyHeight(SerializedProperty property, GUIContent label)
+        VectorPropertyGUIData GetGUIData(SerializedProperty property)
         {
             VectorPropertyGUIData guiData;
             if (!m_GUIDataPerPropertyPath.TryGetValue(property.propertyPath, out guiData))
             {
                 guiData = new VectorPropertyGUIData(GetVectorProperty(property));
                 m_GUIDataPerPropertyPath[property.propertyPath] = guiData;
             }
-            return guiData.Valid ? base.GetPropertyHeight(property, label) : EditorGUIUtility.singleLineHeight;
+            return guiData;
+        }
+
+        public override float GetPropertyHeight(SerializedProperty property, GUIContent label)
+        {
+            return GetGUIData(property).Valid ? base.GetPropertyHeight(property, label) : EditorGUIUtility.singleLineHeight;
         }
 
         public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
         {
-            var guiData = m_GUIDataPerPropertyPath[property.propertyPath];
+            var guiData = GetGUIData(property);
```

Comment:
This call ensures drawing the property works if GetPropertyHeight() has not been called yet